pycheribuild/projects/cross/crosscompileproject.py

```diff
@@ -50,7 +50,8 @@
 # avoids repeating this for every target than can be cross-built
 class CrossCompileMixin(object):
     do_not_add_to_targets = True
-    supported_architectures = CompilationTargets.ALL_SUPPORTED_CHERIBSD_AND_HOST_TARGETS
+    supported_architectures = (CompilationTargets.ALL_SUPPORTED_CHERIBSD_AND_HOST_TARGETS +
```

Comment:
This should be a separate commit, since it adds all the -morello targets.

Reply:
Also you should probably update ALL_SUPPORTED_CHERIBSD_AND_HOST_TARGETS instead.